site/_includes/primary-nav-items.html

```diff
@@ -10,6 +10,6 @@
     <li class="training-menu"><a href="/news/">News</a>
     </li>
     <li>
-        <button class="button-default button-green trigger-top-form"> <a href="https://github.com/apache/incubator-metron/releases" target="new">Download </a></button>
+        <button class="button-default button-green trigger-top-form"> <a href="https://dist.apache.org/repos/dist/release/incubator/metron/" target="new">Download </a></button>
```

Comment:
@cestella Thanks, I didn't see METRON-600. Is it worth pursuing this PR? My thought is yes, so that master is in sync with what is deployed to asf-site. Otherwise, committing #382 to asf-site will revert your changes to the download link.